scripts/rules.go

```diff
@@ -151,6 +151,28 @@ func HttpAPIErrorMessage(m dsl.Matcher) {
 		Report("Field \"Message\" should be a proper sentence with a capitalized first letter and ending in punctuation. $m")
 }
 
+// HttpAPIReturn will report a linter violation if the http function is not
+// returned after writing a response to the client.
+func HttpAPIReturn(m dsl.Matcher) {
+	m.Import("github.com/coder/coder/coderd/httpapi")
+
+	// Manually enumerate the httpapi function rather then a 'Where' condition
+	// as this is a bit more efficient.
+	m.Match(`
+	if err != nil {
```

Comment:
Should we handle optional error checks like the following?
```go
if err != nil && !xerrors.Is(err, sql.ErrNoRows) {
    ...
}
```

Reply:
@AbhineetJain switched it to `if $*_ {` which catches the statement you provided above :+1: